react/components/GalleryItem.tsx

```diff
@@ -32,12 +35,13 @@ function GalleryItem({
   listName,
   summary,
   CustomSummary,
+  preferredSKU
 }: GalleryItemProps) {
   const { push } = usePixel()
   const { searchQuery } = useSearchPage()
 
   const product = useMemo(
-    () => ProductSummary.mapCatalogProductToProductSummary(item),
+    () => ProductSummary.mapCatalogProductToProductSummary(item, preferredSKU),
```

Comment:
Shouldn't `preferredSKU` be added to the dependency array for this `useMemo` call?

Reply:
Added